### DISCUSSION TITLE
Help [possible bug]

### DISCUSSION THREAD

**Discussion by RafTeog** [LBCrion/sfwbar#233] [General]
Greetings, everyone.

I've been loving to use this bar in my sway setup, even if not for that long (3 weeks). Situation: whenever I try to put a background on (swaybg, with standard bg command) the bar disappears. It is still there, I clicked on the empty space, and the terminal poped up, because I had the foot terminal button there. It is literally that line, because whenever I comment it, the bar is there, with normal functions and everything.
I want a system with the major components written in C to test performance and everything . This was almost the only bar that I found (the only other was yambar), thank you very much, I am learning a lot by reading its code and tinkering with the config.

Cheers.

**Comment by LBCrion** [comment]
This sounds like the background is being displayed on a layer over sfwbar.
Can I check which layer you're running sfwbar on? (This is usually set via
SetLayer action or layer = property in the latest git version).

**Reply by RafTeog** [reply]
SetLayer "background". Btw, I am using the sfwbar, not sfwbar-git. Indeed I put 'overlay' and it is now visible.Thx, cheers!
